src/main/cpp/subsystems/Pnenumatics.cpp

```diff
@@ -21,6 +21,12 @@ Pnenumatics::Pnenumatics(){
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
     SetSubsystem("Pnenumatics");

```

Comment:
Really, "pneumatics" should not be a subsystem. Each part of the robot (arm, claw) needs to be a different subsystem. If you think about it, creating one subsystem for pneumatics would be like creating one subsystem for "electrical", even though the electric motors are being used for different things on the robot.